.github/pull_request_template.md

```diff
@@ -0,0 +1,14 @@
+### ✨ Summary
```

Comment:
`[non-blocking question]`

Should we add a placeholder for potentially resolving an issue e.g. `Resolves: #` and let the user complete the issue number?